### ISSUE TITLE
Get full structure object from sourceFile

### ISSUE THREAD

**Issue by anthonybriand** [dsherret/ts-morph#577]
**Describe the bug**

Version: 1.1.0

**To Reproduce**

```ts
import { Project, ts } from "ts-morph";

console.log(ts.version); // 3.3.3

const project = new Project();
const sourceFile = project.addExistingSourceFile("test.ts");

// reproduce the problem here
console.log(sourceFile.getStructure()); // {"bodyText": "export class UpdateFile {\n\n}"}
```

**Expected behavior**

Return a full SourceFileStructure object, actually only return a simple object containing a bodyText key

**Comment by dsherret** [comment]
It's your lucky week. This is resolved by #569 and will be available in 2.0.0 (I will do a release either this weekend or next weekend).

Previously it wasn't possible to get an accurate structure object from a source file because order is important and the source file may contain statements that don't have a corresponding structure. That's why it previously only returned a string of the body text. Note that this involves removing properties like `classes`, `enums` etc from the structure.

**Comment by dsherret** [comment]
I'm going to close this as it's resolved by #569.
